From 3469830e3d95c6176fb91017463c5ef4140dd1c0 Mon Sep 17 00:00:00 2001
From: Nicolas Goaziou <mail@nicolasgoaziou.fr>
Date: Sat, 17 Dec 2016 11:38:53 +0100
Subject: [PATCH 46/50] ox-odt: Fix typo

* lisp/ox-odt.el (org-odt--translate-image-links): Fix typo.
---
 lisp/ox-odt.el | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lisp/ox-odt.el b/lisp/ox-odt.el
index 92c54cf8d..dc2e21663 100644
--- a/lisp/ox-odt.el
+++ b/lisp/ox-odt.el
@@ -3691,7 +3691,7 @@ contextual information."
 ;;; Images
 
 (defun org-odt--translate-image-links (data _backend info)
-  (org-export-insert-image-links data info inline image rules))
+  (org-export-insert-image-links data info org-odt-inline-image-rules))
 
 ;;;; LaTeX fragments
 
-- 
2.13.2

